packages/contracts-bedrock/test/StandardBridge.t.sol

```diff
@@ -14,7 +14,7 @@ import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 ///         internal functions so they can be more easily tested directly.
 contract StandardBridgeTester is StandardBridge {
     constructor(address payable _messenger, address payable _otherBridge)
-        StandardBridge(_messenger, _otherBridge)
+        StandardBridge(StandardBridge(_otherBridge))
```

Comment:
ibid, can we just do
```suggestion
        StandardBridge(_otherBridge)
```

Reply:
The constructor was updated to accept a `StandardBridge`? This type conversion is required right? `otherBridge` is an address